  main.cpp
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\ucrt\crtdbg.h(303,1): warning C4005: '_malloca': redefinición de macro
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\ucrt\malloc.h(122): message : vea la definición anterior de '_malloca'
     Creando biblioteca C:\Uni\Segundo\Segundo cuatri\Progra\TPV2\bin\TPV2Debug.lib y objeto C:\Uni\Segundo\Segundo cuatri\Progra\TPV2\bin\TPV2Debug.exp
main.obj : error LNK2019: símbolo externo SDL_CreateWindow sin resolver al que se hace referencia en la función "void __cdecl firstTest(void)" (?firstTest@@YAXXZ)
main.obj : error LNK2019: símbolo externo SDL_DestroyWindow sin resolver al que se hace referencia en la función "void __cdecl firstTest(void)" (?firstTest@@YAXXZ)
main.obj : error LNK2019: símbolo externo SDL_CreateRenderer sin resolver al que se hace referencia en la función "void __cdecl firstTest(void)" (?firstTest@@YAXXZ)
main.obj : error LNK2019: símbolo externo SDL_SetRenderDrawColor sin resolver al que se hace referencia en la función "void __cdecl firstTest(void)" (?firstTest@@YAXXZ)
main.obj : error LNK2019: símbolo externo SDL_RenderClear sin resolver al que se hace referencia en la función "void __cdecl firstTest(void)" (?firstTest@@YAXXZ)
main.obj : error LNK2019: símbolo externo SDL_RenderPresent sin resolver al que se hace referencia en la función "void __cdecl firstTest(void)" (?firstTest@@YAXXZ)
main.obj : error LNK2019: símbolo externo SDL_DestroyRenderer sin resolver al que se hace referencia en la función "void __cdecl firstTest(void)" (?firstTest@@YAXXZ)
main.obj : error LNK2019: símbolo externo SDL_Delay sin resolver al que se hace referencia en la función "void __cdecl firstTest(void)" (?firstTest@@YAXXZ)
main.obj : error LNK2019: símbolo externo SDL_Init sin resolver al que se hace referencia en la función "void __cdecl firstTest(void)" (?firstTest@@YAXXZ)
main.obj : error LNK2019: símbolo externo SDL_Quit sin resolver al que se hace referencia en la función "void __cdecl firstTest(void)" (?firstTest@@YAXXZ)
MSVCRTD.lib(exe_main.obj) : error LNK2019: símbolo externo main sin resolver al que se hace referencia en la función "int __cdecl invoke_main(void)" (?invoke_main@@YAHXZ)
C:\Uni\Segundo\Segundo cuatri\Progra\TPV2\SDL2-2.0.7\lib\x86\SDL2.lib : warning LNK4272: el tipo de máquina de biblioteca 'x86' está en conflicto con el tipo de máquina de destino 'x64'
C:\Uni\Segundo\Segundo cuatri\Progra\TPV2\SDL2-2.0.7\lib\x86\SDL2main.lib : warning LNK4272: el tipo de máquina de biblioteca 'x86' está en conflicto con el tipo de máquina de destino 'x64'
C:\Uni\Segundo\Segundo cuatri\Progra\TPV2\SDL2_image-2.0.1\lib\x86\SDL2_image.lib : warning LNK4272: el tipo de máquina de biblioteca 'x86' está en conflicto con el tipo de máquina de destino 'x64'
C:\Uni\Segundo\Segundo cuatri\Progra\TPV2\SDL2_ttf-2.0.14\lib\x86\SDL2_ttf.lib : warning LNK4272: el tipo de máquina de biblioteca 'x86' está en conflicto con el tipo de máquina de destino 'x64'
C:\Uni\Segundo\Segundo cuatri\Progra\TPV2\SDL2_mixer-2.0.2\lib\x86\SDL2_mixer.lib : warning LNK4272: el tipo de máquina de biblioteca 'x86' está en conflicto con el tipo de máquina de destino 'x64'
C:\Uni\Segundo\Segundo cuatri\Progra\TPV2\SDL2_net-2.0.1\lib\x86\SDL2_net.lib : warning LNK4272: el tipo de máquina de biblioteca 'x86' está en conflicto con el tipo de máquina de destino 'x64'
C:\Uni\Segundo\Segundo cuatri\Progra\TPV2\bin\TPV2Debug.exe : fatal error LNK1120: 11 externos sin resolver
